### ISSUE TITLE
A11y: [Start a call] Participant count icon focused 3x; lacks context....

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-ios#5245]
- **Checkpoint ID:** 11.1.1.1b/12.4
- **View/Component:** Start a call

### Description
Participant count icon focused 3x; lacks context.

### Recommendation
Ensure single focus stop and add 'Participants in call' label.

--- 
*Report Date: 20.02.2026*
